apps/core/app/[locale]/(default)/login/_actions/submit-reset-password-form.ts

```diff
@@ -0,0 +1,44 @@
+'use server';
+
+import { z } from 'zod';
+
+import { ResetPasswordSchema, submitResetPassword } from '~/client/mutations/submit-reset-password';
+
+interface SubmitResetPasswordForm {
+  formData: FormData;
+  path: string;
+  reCaptchaToken: string;
+}
+
+export const submitResetPasswordForm = async ({
+  formData,
+  path,
+  reCaptchaToken,
+}: SubmitResetPasswordForm) => {
+  try {
+    const parsedData = ResetPasswordSchema.parse({
+      email: formData.get('email'),
+    });
+
+    const response = await submitResetPassword({
+      email: parsedData.email,
+      path,
+      reCaptchaToken,
+    });
+
+    if (response.customer.requestResetPassword.errors.length === 0) {
+      return { status: 'success', data: parsedData };
+    }
+
+    return {
+      status: 'failed',
```

Comment:
should we change `status` to `error` here?

Reply:
Hmm, I was sure that I did change it. Thanks for catching this.